src/translate/pddl/tasks.py

```diff
@@ -68,15 +68,22 @@ def dump(self):
             for axiom in self.axioms:
                 axiom.dump()
 
+
+REQUIREMENT_LABELS = [
```

Comment:
Since Python 3.8, there's a notion of `Final`: https://peps.python.org/pep-0591/